Dockerfiles/ci/Dockerfile

```diff
@@ -10,7 +9,6 @@ RUN export ARCH=$(case $(arch) in x86_64) echo -n amd64 ;; aarch64) echo -n arm6
     mkdir /project/output && \
     dnf install --setopt=install_weak_deps=False -y git-core ansible python3-pip mlocate && \
     dnf clean all && \
-    pip3 install operator-courier && \
-    git clone --branch $OPERATOR_TEST_PLAYBOOKS_TAG https://github.com/redhat-operator-ecosystem/operator-test-playbooks.git
-COPY ./entrypoint_cvp.sh /project/entrypoint_cvp.sh
-ENTRYPOINT ["/project/entrypoint_cvp.sh"]
+    pip3 install operator-courier
+COPY ./entrypoint.sh /project/entrypoint.sh
+ENTRYPOINT ["/project/entrypoint.sh"]
```

Comment:
I am sorry but I still do not think that ENTRYPOINT is ideal here.

Sure you can override it with podman run --entrypoint.
But we will like to run more tests in future.

The current version of "test" expect execution as part of "entrypoint" and assertions are executed later.
that will complicate running more independent tests with the same entrypoint.
And you can have just single entrypoint in dokcerfile

https://docs.docker.com/engine/reference/builder/#entrypoint

https://docs.docker.com/engine/reference/builder/#cmd

Reply:
@lslebodn So you would like to have all the assertions also as a part of the container run such that 
the container ran locally would run the tests and assertions together.